README.md

```diff
@@ -116,6 +117,19 @@ An array of package names needed for httpd Phusion Passenger support.  The defau
 
 This class manages the Apache web server to provide HTTPS support.
 
+
+#### apache::mod\_status class
+
+This class manages the Apache status module which provides reports of server activity.  By default access will
+only be allowed on the local loopback interface.
+
+##### `location`
+This parameter controls the location used to access the apache status page.  Default value is set to /server-status.
+
+##### `allow_from`
+An array of IPs or IP ranges in CIDR format which define what hosts are allowed to access the server status page.
```

Comment:
Does this have a default?  If not, please annotate as `(required)` like I've done elsewhere, again for consistency.